src/config/db.ts

```diff
@@ -1,19 +1,27 @@
 import { MongoClient } from 'mongodb';
+import dotenv from 'dotenv';
 
-// URL de conexión a MongoDB Atlas
-export const uri = "mongodb+srv://Isa_tm_22:1234@cluster-qbit.pb6em.mongodb.net/Qbit-Test?retryWrites=true&w=majority";
+dotenv.config();
+// Get the password from the environment variable
+const password = process.env.MONGO_PASSWORD;
+const user = process.env.MONGO_USER;
+const database = process.env.MONGO_DATABASE;
 
-// Crear una nueva instancia de MongoClient
+// MongoDB Atlas connection URL using the environment variable
+export const uri = `mongodb+srv://${user}:${password}@cluster-qbit.pb6em.mongodb.net/${database}?retryWrites=true&w=majority`;
```

Comment:
Usar la conexion directa de la .env, para no exponer partes de la uri

```suggestion
export const uri = process.env.DB_URI;
```